### DISCUSSION TITLE
How many signs are in the 🇨🇳📅?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#758] [Galaxy Trivia] [answered]
*List them all as emojis.*

**Comment by peterbaccaro** [comment, marked as answer]
There are **12** Chinese zodiac signs, in the following order:

🐀 - Rat
🐂 - Ox
🐯 - Tiger
🐰 Rabbit
🐉 - Dragon
🐍 - Snake
🐴 - Horse
🐐 - Goat
🐒 - Monkey
🐓 - Rooster
🐶 - Dog
🐷 - Pig
